pkgs/development/libraries/glib/default.nix

```
@@ -154,11 +161,25 @@ stdenv.mkDerivation rec {
     patchShebangs glib/gen-unicode-tables.pl
     patchShebangs glib/tests/gen-casefold-txt.py
     patchShebangs glib/tests/gen-casemap-txt.py
```

```suggestion
    patchShebangs glib/gen-unicode-tables.pl \
      glib/tests/gen-casefold-txt.py \
      glib/tests/gen-casemap-txt.py
```
Unrelated but we can simplify this

---

This PR is big enough as it is, so I don't want to start adding in unrelated changes.